Error in opening GPX files / pocket queries in current nightly version

### Describe your problem!

A couple of users on support report that they cannot open a GPX file or pocket query any longer, c:geo stops for them.

Received a log file today:
```
01-08 11:00:35.766 11113 11113 E AndroidRuntime: java.lang.RuntimeException: Unable to start activity ComponentInfo{cgeo.geocaching/cgeo.geocaching.CacheListActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String java.lang.String.toLowerCase(java.util.Locale)' on a null object reference
01-08 11:00:35.766 11113 11113 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4166)
01-08 11:00:35.766 11113 11113 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4312)
01-08 11:00:35.766 11113 11113 E AndroidRuntime: 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:101)
01-08 11:00:35.766 11113 11113 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
01-08 11:00:35.766 11113 11113 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
01-08 11:00:35.766 11113 11113 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2571)
01-08 11:00:35.766 11113 11113 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
01-08 11:00:35.766 11113 11113 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:226)
01-08 11:00:35.766 11113 11113 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:313)
01-08 11:00:35.766 11113 11113 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8741)
01-08 11:00:35.766 11113 11113 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
01-08 11:00:35.766 11113 11113 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
01-08 11:00:35.766 11113 11113 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1067)
01-08 11:00:35.766 11113 11113 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String java.lang.String.toLowerCase(java.util.Locale)' on a null object reference
01-08 11:00:35.766 11113 11113 E AndroidRuntime: 	at cgeo.geocaching.CacheListActivity.onCreate(CacheListActivity.java:498)
01-08 11:00:35.766 11113 11113 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:8578)
01-08 11:00:35.766 11113 11113 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:8557)
01-08 11:00:35.766 11113 11113 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1384)
01-08 11:00:35.766 11113 11113 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4147)
01-08 11:00:35.766 11113 11113 E AndroidRuntime: 	... 12 more
```



### How to reproduce?

- try to load a pocket query
- or open an attached GPX file from email

### Actual result after these steps?

c:geo crashes as shown in logfile above

### Expected result after these steps?

import of GPX file succeeds

### Reproducible

Yes

### c:geo Version

2023.01.08-NB

### System information

_No response_

### Additional Information

probably caused by change in #13788

--- a/main/src/main/java/cgeo/geocaching/CacheListActivity.java
+++ b/main/src/main/java/cgeo/geocaching/CacheListActivity.java
@@ -495,7 +495,7 @@ public class CacheListActivity extends AbstractListActivity implements FilteredA
                 listNameMemento.rememberTerm(extras.getString(Intents.EXTRA_NAME));
             } else {
                 final String data = ContentStorage.get().getName(getIntent().getData());
-                listNameMemento.rememberTerm(StringUtils.endsWith(data.toLowerCase(Locale.ROOT), ".gpx") ? StringUtils.substring(data, 0, -4) : data);
+                listNameMemento.rememberTerm(StringUtils.isNotBlank(data) && StringUtils.endsWith(data.toLowerCase(Locale.ROOT), ".gpx") ? StringUtils.substring(data, 0, -4) : data);
             }
             importGpxAttachement();
         }

